velox/connectors/hive/storage_adapters/abfs/AbfsWriteFile.cpp

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/connectors/hive/storage_adapters/abfs/AbfsWriteFile.h"
+#include <azure/storage/files/datalake.hpp>
```

Comment:
nit: empty line above.

Reply:
We need to add a blank line between the various inlude categories. See 
https://github.com/facebookincubator/velox/discussions/7723#discussioncomment-7687271

Reply:
oh, got it! thank you @majetideepak . Resolved